### ISSUE TITLE
move variables to environment file

### ISSUE THREAD

**Issue by joicemjoseph** [janastu/mediastore#2]
0596b4a  mediastore/mediastore/config.yaml#L4 contains secret keys. 
is that good practice? is that necessary to move those variables to an env file ?

**Comment by arunk** [comment]
Good point, those should be moved to a file not tracked in git. And an example conf file provided which is committed to git.
